exercises/practice/binary-search/.meta/example.jl

```diff
@@ -1,4 +1,7 @@
 # the base function searchsorted does exactly what the exercise asks for
 const binarysearch = searchsorted
 
-enable_bonus_tests = true
+enable_reverse_tests = true
+enable_by_tests = true
+enable_lt_tests = true
+enable_multiple_matches = true
```

Comment:
I like the idea but maybe it could be a dict or named tuple that maps `:rev, :by, ...` to booleans to make it more obvious?

Reply:
Sure. We could leave this in uncommented:

```jl
# Change `false` to `true` for each set of bonus tests you want enabled
enabled_bonus_tests = (rev = false, by = false, lt = false, multiple_matches = false)
```

Reply:
Just an idea here but wouldn't make the person doing the exercise have to comment out/write more lines either with the tuple or the Dict? I mean if one writes it as:
```
bonus_tests = Dict(
#     :by => true,
#    :lt => true,
#    :rev => true,
#    :multiple_matches => true
# )
```

Reply:
But four separate flags is also fine by me and may be easier for people to understand.

I don't mind which is used. I leave it up to you two to decide something.

Reply:
Sorry @gpucce, I'm not meaning to steal your thunder :)

Reply:
Oh nevermind at all!
This is very small PR I am happy to learn a bit how a better "interface" would look.
And time-wise any choice will be a very small effort!

Reply:
I like Colin's suggestion but have no strong opinion on it either. Let's go with that if you're happy with it as well and then we can merge it :)

Reply:
Sure! will commit shortly

Reply:
You'll probably want to use something like `@isdefined(enabled_bonus_tests) && get(enabled_bonus_tests, :rev, false)`. To avoid key errors. Maybe that's obvious?

Reply:
Would have done smth like this but I would have asked for confirmation, thanks!